netbout-web/src/test/java/com/netbout/rest/bout/TkEmVerifyTest.java

```diff
@@ -113,7 +117,36 @@ public void rejectsUnneededVerification() throws Exception {
             request("urn:test:3:alias3:new3@example.com")
         );
     }
-
+    /**
+     *
+     * TkEmVerify can return user friendly message if
+     * the verification url is not properly encoded.
+     * @throws Exception If some problem inside
+     */
+    @Test
+    public void returnsUserFriendlyMessage() throws Exception {
+        final MkBase base = new MkBase();
+        final String urn = "urn:test:4";
+        final User user = base.user(new URN(urn));
+        user.aliases().add("alias4");
+        final Alias alias = user.aliases().iterate().iterator().next();
+        alias.email("old@example.com!new4@example.com");
+        try {
+            new TkEmVerify(base).act(
+                    new RqRegex.Fake(PATTERN, "x")
+            );
+        } catch (final RsForward ex) {
+            final Iterator<String> response = ex.head().iterator();
```

Comment:
@kujtimiihoxha I would also add an assert: `Matcher.assertThat(ex, Matchers.not(Matchers.instanceOf(RsFailure.class)));`